src/pullRequestDescription.md

```diff
@@ -25,6 +25,15 @@ git pull origin {{base}}
 git merge {{head}}
 ```
 
+Now resolve the conflicts locally.
+
+*Please take care that these changes actually land in `{{base}}` as quickly as possible, so the next person
+that needs to merge changes from `{{head}}` to `{{base}}` does not encounter the same conflicts that you just resolved.*
+
+Depending on the repository and target branch, direct pushes to `{{base}}` might be forbidden. In that case, you need to
```

Comment:
Shall we also provide the exact shell commands for this case?

Reply:
Can do.

Reply:
Done